docs/index.html

```
@@ -1,2 +1,2 @@
 <!-- Redirect to latest version -->
-<meta HTTP-EQUIV="REFRESH" content="0; url=./4.4.0/index.html">
+<meta HTTP-EQUIV="REFRESH" content="0; url=./4.4.1/index.html">
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)